corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -211,6 +216,7 @@ def build_remote_request_queries(self):
                 data=self._remote_request_query_datums,
                 prompts=self.build_query_prompts(),
                 default_search=self.module.search_config.default_search,
+                results_title=self.build_results_title() if self.module.search_config.command_label != {} else None,
```

Comment:
would `self.module.search_config.command_label != {}` ever be true given it has a [default value](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L2110) ?

Reply:
good point changed https://github.com/dimagi/commcare-hq/pull/33261#:~:text=a%20default%20value-,e253da9,-Merge%20state